lib/coap/socket_server.ex

```diff
@@ -128,7 +122,7 @@ defmodule CoAP.SocketServer do
     :telemetry.execute(
       [:coap_ex, :connection, :data_sent],
       %{size: byte_size(data)},
-      %{host: ip, port: port}
+      %{host: ip, port: port, tag: tag}
```

Comment:
@johngiffin You mentioned maybe adding the `token`, too. Is that still in the plan?